LICENSE.md

```diff
@@ -93,21 +90,21 @@ A Derivative must contain changes which a reasonably informed person would consi
 The User may distribute a Derivative if the following conditions are met:
  - The Derivative does not generate revenue.
  - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+ - The Derivative, as a whole, is licensed under this license for anyone who comes into possession of a copy.
```

Comment:
There's part of your problem. Stop trying to copy other licenses. This license isn't other licenses.

"Complies" means to "act in accordance with a wish or command". Eg, "license compliance" means "acting in accordance with the license"